packages/dashboard-frontend/src/services/factory-location-adapter/index.ts

```diff
@@ -52,7 +52,7 @@ export class FactoryLocationAdapter implements FactoryLocation {
   }
 
   public static isHttpLocation(href: string): boolean {
-    return /^(https?:\/\/.)[-a-zA-Z0-9@:%._+~#=]{2,}\b([-a-zA-Z0-9@:%_+.~#?&/=]*)$/.test(href);
+    return /^(https?:\/\/.)[-a-zA-Z0-9@:%._+~#=]{2,}\b([-a-zA-Z0-9@:%_+.~#?&/={},]*)$/.test(href);
```

Comment:
The `{` and `}` characters are not considered valid for URIs, but since they can still exist in the window.location object, I propose to add it to this regex

Reply:
@dkwon17 Would you be able to add an appropriate test?

Reply:
Yes, my latest commits adds a few tests, WDYT?